owncloudData/src/test/java/com/owncloud/android/data/files/repository/OCFileRepositoryTest.kt

```diff
@@ -206,6 +210,36 @@ class OCFileRepositoryTest {
         }
     }
 
+    @Test
+    fun `getFilesLastUsageIsOlderThanGivenTime returns a list of OCFile`() {
+        every {
+            localFileDataSource.getFilesLastUsageIsOlderThanGivenTime(timeInMilliseconds)
+        } returns emptyList()
+
+        val result = ocFileRepository.getFilesLastUsageIsOlderThanGivenTime(timeInMilliseconds)
+
+        assertEquals(emptyList<OCFile>(), result)
+
+        verify(exactly = 1) {
+            localFileDataSource.getFilesLastUsageIsOlderThanGivenTime(timeInMilliseconds)
+        }
+    }
+
+    @Test
+    fun `getFilesLastUsageIsOlderThanGivenTime returns an empty list when datasource returns an empty list`() {
+        every {
+            localFileDataSource.getFilesLastUsageIsOlderThanGivenTime(timeInMilliseconds)
+        } returns listOf(OC_FILE_AVAILABLE_OFFLINE)
+
+        val result = ocFileRepository.getFilesLastUsageIsOlderThanGivenTime(timeInMilliseconds)
+
+        assertEquals(listOf(OC_FILE_AVAILABLE_OFFLINE), result)
+
+        verify(exactly = 1) {
+            localFileDataSource.getFilesLastUsageIsOlderThanGivenTime(timeInMilliseconds)
+        }
+    }
```

Comment:
You confused the content of these 2 tests. And the same as in the data source tests, these methods shouldn't return av. offline files

Reply:
Done